style-editor.scss

```diff
@@ -55,6 +55,21 @@ body {
 	}
 }
 
+/** === Editor Changes === */
```

Comment:
Can we make this a little more descriptive? It's a little vague right now. Maybe "Editor Toolbar Position" or something similar.